tests/wakunode2/test_app.nim

```diff
@@ -53,3 +56,29 @@ suite "Wakunode2 - App initialization":
 
     ## Then
     check res.isOk()
+
+  test "node setup is successful":
```

Comment:
I cannot use `asyncTest` here due to some non-GC-safe issue. I am using `waitFor` as a temporary workaround.